messagingpush/src/main/java/io/customer/messagingpush/CustomerIOPushNotificationHandler.kt

```diff
@@ -168,13 +168,9 @@ internal class CustomerIOPushNotificationHandler(
             .setStyle(NotificationCompat.BigTextStyle().bigText(body))
         tintColor?.let { color -> notificationBuilder.setColor(color) }
 
-        // This is custom logic to add images to a push notification. Some image URLs have not
-        // been able to display an image in a push. https://github.com/customerio/issues/issues/7293
-        // Instead, we recommend customers use `notification.image` in the FCM payload to bypass
-        // our logic and use the logic from FCM's SDK for image handling instead.
         try {
-            // check for image
-            val notificationImage = bundle.getString(IMAGE_KEY)
+            // check for image in data and notification payload
```

Comment:
Could you add a comment explaining why we are checking for an image in data and in notification payload? So future readers of this code understand the decision behind doing this.